packages/pn-pa-webapp/src/navigation/__test__/RequireAuth.test.tsx

```diff
@@ -13,11 +14,15 @@ jest.mock('react-router-dom', () => {
   };
 });
 
-jest.mock('@pagopa-pn/pn-commons', () => {
-  const original = jest.requireActual('@pagopa-pn/pn-commons');
+// Unfortunately a mock on SessionModal won't work since it is invoked from HandleAuth, which also lies in pn-commons.
+// We mock the Dialog which is used inside SessionModal instead.
+// --------------------
+// Carlos Lombardi, 2022.08.18
+jest.mock('@mui/material', () => {
```

Comment:
Il mock sul SessionModal che errore dava?

Reply:
Non aveva nessun effetto, perchè dentro `HandleAuth` l'import di `SessionModal` è diretto su un file, facendo parte tutte due i componenti del package pn-commons. Questo l'ho descritto nel commentone in inglese, c'è una sottosezione "Tests" dentro "What I did ..."